src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final int amountFigures = 6;
+        final int half = 3;
```

Comment:
The constant `half` is not self-explanatory, and its purpose is not clear from the name. Consider renaming it to reflect its usage more clearly, such as `HALF_FIGURES_COUNT`, and ensure it's calculated based on `FIGURES_COUNT` to avoid magic numbers.